community/contributing/updating_the_class_reference.rst

```diff
@@ -154,220 +179,41 @@ repository:
-+----------------------------+--------------------------------------+-----------------------------------+---------------------------------------------------+
-| [codeblocks] [/codeblocks] | [codeblock] for multiple languages   | *See below.*                      | *See below.*                                      |
-+----------------------------+--------------------------------------+-----------------------------------+---------------------------------------------------+
-| [gdscript] [/gdscript]     | GDScript codeblock tab in codeblocks | *See below.*                      | *See below.*                                      |
-+----------------------------+--------------------------------------+-----------------------------------+---------------------------------------------------+
-| [csharp] [/csharp]         | C# codeblock tab in codeblocks       | *See below.*                      | *See below.*                                      |
-+----------------------------+--------------------------------------+-----------------------------------+---------------------------------------------------+
-
-Use ``[codeblock]`` for pre-formatted code blocks. Inside ``[codeblock]``, always use **four spaces** for indentation (the parser will delete tabs). Example:
-
-.. code-block:: none
-
-    [codeblock]
-    func _ready():
-        var sprite = get_node("Sprite")
-        print(sprite.get_pos())
-    [/codeblock]
-
-Will display as:
-
-.. code-block:: gdscript
-
-    func _ready():
-        var sprite = get_node("Sprite")
-        print(sprite.get_pos())
-
-If you need to have different code version in GDScript and C#, use ``[codeblocks]`` instead. If you use ``[codeblocks]``, you also need to have at least one of the language specific tags (``[gdscript]`` and ``[csharp]``). Always write GDScript code examples first! You can use this `experimental code translation tool <https://github.com/HaSa1002/codetranslator>`_ to speed up your workflow.
-
-.. code-block:: none
-
-    [codeblocks]
-    [gdscript]
-    func _ready():
-        var sprite = get_node("Sprite")
-        print(sprite.get_pos())
-    [/gdscript]
-    [csharp]
-    public override void _Ready()
-    {
-        var sprite = GetNode("Sprite");
-        GD.Print(sprite.GetPos());
-    }
-    [/csharp]
-    [/codeblocks]
-
-Will display as:
-
-.. tabs::
- .. code-tab:: gdscript GDScript
-
-    func _ready():
-        var sprite = get_node("Sprite")
-        print(sprite.get_pos())
-
- .. code-tab:: csharp
-
-    public override void _Ready()
-    {
-        var sprite = GetNode("Sprite");
-        GD.Print(sprite.GetPos());
-    }
+To learn to create a pull request, read `Creating a pull request
+<https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request>`_
+in the GitHub documentation.
 
+.. warning::
```

Comment:
Suggestion to replace the text of the whole warning (feel free to adapt further, I just felt that the current one focuses too much on less relevant aspects):

```
Unless doing trivial changes (e.g. typo fixes), we do not recommend using the GitHub web editor to edit the class reference XML.
Its features to edit XML are limited (e.g. to keep indentation consistent), and it does not allow amending commits based on feedback from reviewers. Moreover, you wouldn't be able to test your changes in the engine or with validation scripts as described in <add reference to class_reference_writing_guidelines.rst::How to edit class XML section>.
```